### PR TITLE
Clean up grammar and consistency in documentation

### PR DESCRIPTION
**Pull request by deepyaman**
## Notice

- [x] I acknowledge and agree that, by checking this box and clicking "Submit Pull Request":

- I submit this contribution under the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0.txt) and represent that I am entitled to do so on behalf of myself, my employer, or relevant third parties, as applicable.
- I certify that (a) this contribution is my original creation and / or (b) to the extent it is not my original creation, I am authorised to submit this contribution on behalf of the original creator(s) or their licensees.
- I certify that the use of this contribution as authorised by the Apache 2.0 license does not violate the intellectual property rights of anyone else.

## Motivation and Context
Why was this PR created?

## How has this been tested?
What testing strategies have you used?

## Checklist

- [x] Read the [contributing](/CONTRIBUTING.md) guidelines
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [ ] Added a description of this change and added my name to the list of supporting contributions in the [`RELEASE.md`](/RELEASE.md) file
- [ ] Added tests to cover my changes
- [ ] Assigned myself to the PR

### CODE REVIEW

File: docs/source/05_resources/05_faq.md [hunk 1]
```diff
@@ -16,8 +16,8 @@ As we see it, CausalNex:
 
 - **Generates transparency and trust in models** it creates by allowing users to collaborate with domain experts during the modelling process.
 - Uses an **optimised structure learning algorithm**, [NOTEARS](https://papers.nips.cc/paper/8157-dags-with-no-tears-continuous-optimization-for-structure-learning.pdf) where the runtime to learn structure is no longer exponential but scales cubically with number of nodes.
-- **Add known relationships or remove spurious correlations** so that your model can better consider causal relationships in data
-- **Visualise networks using common tools** built upon [NetworkX](https://networkx.github.io/), allowing users to understand relationships in their data more intuitively, and work with experts to encode their knowledge
+- **Adds known relationships or remove spurious correlations** so that your model can better consider causal relationships in data.
```
Comment (pjbqb) [inline docs/source/05_resources/05_faq.md]: This sort of suggests it's automatic, which is incorrect. "Ability to add" would be better than "Adds" I think

Comment (deepyaman) [reply docs/source/05_resources/05_faq.md]: Reworded!

File: docs/source/05_resources/05_faq.md [hunk 2]
```diff
@@ -88,7 +88,7 @@ Once CausalNex is installed, you can check your version as follows:
 pip show causalnex
 ```
 
-To later upgrade CausalNex to a different version, simply run:
+To later upgrade CausalNex to a different version, suimply run:
```
Comment (benhorsburgh) [inline docs/source/05_resources/05_faq.md]: Introduced a mistake here!
```suggestion
To later upgrade CausalNex to a different version, simply run:
```

**Comment by deepyaman** [comment]
> Looks good, think there's an accidental keyboard type in there to remove before we can merge...

Thanks! That’s what I get for committing from my phone...

**Pull request merged**